packages/engine/Specs/Scene/Model/ModelSpec.js

```diff
@@ -701,57 +804,52 @@ describe(
     });
 
     it("renders Draco-compressed model", function () {
-      return loadAndZoomToModel({ gltf: dracoCesiumManUrl }, scene).then(
+      return loadAndZoomToModelAsync({ gltf: dracoCesiumManUrl }, scene).then(
         function (model) {
           verifyRender(model, true);
         }
       );
     });
 
-    it("fails to load with Draco decoding error", function () {
-      const readyPromise = pollToPromise(function () {
+    it("fails to load with Draco decoding error", async function () {
+      DracoLoader._getDecoderTaskProcessor();
+      await pollToPromise(function () {
         return DracoLoader._taskProcessorReady;
       });
-      DracoLoader._getDecoderTaskProcessor();
-      return readyPromise
-        .then(function () {
-          const decoder = DracoLoader._getDecoderTaskProcessor();
-          spyOn(decoder, "scheduleTask").and.callFake(function () {
-            return Promise.reject({ message: "Custom error" });
-          });
 
-          const model = scene.primitives.add(
-            Model.fromGltf({
-              url: dracoCesiumManUrl,
-            })
-          );
+      const decoder = DracoLoader._getDecoderTaskProcessor();
```

Comment:
It looks like `DracoLoader._getDecoderTaskProcessor()` should be converted to an async function. (Maybe not for this PR)

Reply:
I agree- Though these PRs are large, and this shouldn't be a significant API change so it doesn't need to go out timed with this batch of deprecations.